code/game/area/areas.dm

```diff
@@ -23,6 +23,7 @@
 									'sound/ambience/ambigen8.ogg','sound/ambience/ambigen9.ogg',\
 									'sound/ambience/ambigen10.ogg','sound/ambience/ambigen11.ogg',\
 									'sound/ambience/ambigen12.ogg','sound/ambience/ambigen14.ogg')
+	var/can_be_dirty = TRUE //Set this to FALSE to prevent an area from being made dirty at all
```

Comment:
defining MORE variables is slower, and a clusterfuck for future coders and admins alike to decipher.

flags explicitly exists for the purpose of boolean toggles. Especially boolean toggles as niche as this one. You aren't iterating through a list of flags. You are using bitwise math to compare them.

3 is (1&2)
if your bitflag is '2' and 'NOSLIP' is '1' then the flag would be '3'
Then when you check if flags & 2 it will return true. 

Because of the way byond stores variables:

bitflags using the flags var:

list/vars = list("flags" = BYTE) where byte is a single byte of data. But you're also storing the text name of the variable.

In the case of defining a new variable explicitly for the boolean:

list/vars = list("flags" = BYTE, "newvar" = BIT) where you're storing an additional X bytes of data just for the new variable definition, even though the variables value is only a single bit of data.

And keep in mind we have a little over a million individual turfs during runtime, and this extra variable is defined for all of them. (though im pretty sure byond only stores information about variables that are not equal to their initial value.)

At face value, it's a very negligible amount of efficiency loss, but it also clogs up VV menus, and reduces code clarity and consistency for the sake of a single bit of data. Which as i understand in this case, is only used once during pre-round setup anyways?

Some other frequently used variables, or variables which are frequently admin-altered might warrant its own variable, but not this one.